[2023-10-31T22:39:52.191+0100] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: la_test.extract_teams_groups __airflow_temporary_run_2023-10-31T17:13:33.488673+00:00__ [queued]>
[2023-10-31T22:39:52.194+0100] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: la_test.extract_teams_groups __airflow_temporary_run_2023-10-31T17:13:33.488673+00:00__ [queued]>
[2023-10-31T22:39:52.194+0100] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-10-31T22:39:52.199+0100] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): extract_teams_groups> on 2023-10-31 17:13:33.488662+00:00
[2023-10-31T22:39:52.202+0100] {standard_task_runner.py:57} INFO - Started process 66182 to run task
[2023-10-31T22:39:52.205+0100] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'la_test', 'extract_teams_groups', '__airflow_temporary_run_2023-10-31T17:13:33.488673+00:00__', '--job-id', '63', '--raw', '--subdir', 'DAGS_FOLDER/test_aj.py', '--cfg-path', '/var/folders/4w/73_t653s0rz1z847_g217g2h0000gn/T/tmpytlghhhh']
[2023-10-31T22:39:52.205+0100] {standard_task_runner.py:85} INFO - Job 63: Subtask extract_teams_groups
[2023-10-31T22:39:52.225+0100] {task_command.py:416} INFO - Running <TaskInstance: la_test.extract_teams_groups __airflow_temporary_run_2023-10-31T17:13:33.488673+00:00__ [running]> on host mbp21-14
[2023-10-31T22:39:52.247+0100] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='nick.volkmann@tu-dresden.de' AIRFLOW_CTX_DAG_OWNER='volkmann' AIRFLOW_CTX_DAG_ID='la_test' AIRFLOW_CTX_TASK_ID='extract_teams_groups' AIRFLOW_CTX_EXECUTION_DATE='2023-10-31T17:13:33.488662+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='__airflow_temporary_run_2023-10-31T17:13:33.488673+00:00__'
[2023-10-31T22:44:33.646+0100] {local_task_job_runner.py:274} WARNING - Recorded pid 66694 does not match the current pid 66182
[2023-10-31T22:44:33.651+0100] {process_utils.py:131} INFO - Sending Signals.SIGTERM to group 66182. PIDs of all processes in the group: [66182]
[2023-10-31T22:44:33.651+0100] {process_utils.py:86} INFO - Sending the signal Signals.SIGTERM to group 66182
[2023-10-31T22:45:33.690+0100] {process_utils.py:149} WARNING - process psutil.Process(pid=66182, name='python3.9', status='running', started='22:39:52') did not respond to SIGTERM. Trying SIGKILL
[2023-10-31T22:45:33.692+0100] {process_utils.py:86} INFO - Sending the signal Signals.SIGKILL to group 66182
[2023-10-31T22:45:33.700+0100] {process_utils.py:79} INFO - Process psutil.Process(pid=66182, name='python3.9', status='terminated', exitcode=<Negsignal.SIGKILL: -9>, started='22:39:52') (66182) terminated with exit code Negsignal.SIGKILL
[2023-10-31T22:45:33.700+0100] {standard_task_runner.py:172} ERROR - Job 64 was killed before it finished (likely due to running out of memory)
